diff --git a/franka_hardware/include/franka_hardware/robot.hpp b/franka_hardware/include/franka_hardware/robot.hpp
index 6f6e5d6..b92c9a3 100644
--- a/franka_hardware/include/franka_hardware/robot.hpp
+++ b/franka_hardware/include/franka_hardware/robot.hpp
@@ -339,8 +339,8 @@ class Robot {
   bool cartesian_pose_low_pass_filter_active_{false};
   bool cartesian_pose_command_rate_limit_active_{false};
 
-  bool joint_position_command_rate_limit_active_{false};
-  bool joint_position_command_low_pass_filter_active_{false};
+  bool joint_position_command_rate_limit_active_{true};
+  bool joint_position_command_low_pass_filter_active_{true};
 
   double low_pass_filter_cut_off_freq{100.0};
 
